src/suggestSimilarCommands.ts

```diff
@@ -0,0 +1,20 @@
+import commandList from './commandHandlers';
+
+// @ts-ignore
+import * as soundex from 'soundex-code';
+
+const maxLength = 2;
+const map = new Map();
+commandList
+  .forEach((commandItem) => map.set(commandItem, soundex(commandItem.triggers[0], maxLength)));
+
+const reverseMap = new Map();
+for (const [key, value] of map.entries()) {
+  const previous = reverseMap.get(value) || [];
+  previous.push(key);
+  reverseMap.set(value, previous);
+}
+
+export const suggestSimilarCommands = (command: string) => {
+  return reverseMap.get(soundex(command, maxLength));
+};
```

Comment:
could you add a few comments or name the variables differently? I was a bit confused about what this block of code did, and perhaps other devs looking at this might not know what is the purpose of `soundex-code`. Here is a small suggestion:
```suggestion
const maxLength = 2;
const commandSoundexMap = new Map();
commandList
  .forEach((commandItem) => commandSoundexMap.set(commandItem, soundex(commandItem.triggers[0], maxLength)));

// This map contains the list of commands for a given soundex key
const reverseMap = new Map<string, CommandHandler[]>();
for (const [commandFunction, soundexValue] of commandSoundexMap.entries()) {
  const previous = reverseMap.get(soundexValue) || [];
  previous.push(commandFunction);
  reverseMap.set(soundexValue, previous);
}

export const suggestSimilarCommands = (command: string) => {
  return reverseMap.get(soundex(command, maxLength));
};
```

Let me know what you think @RaisinTen  😃

Reply:
@BOLT04 hehe, it felt a bit difficult finding a good name for the variables here. Since I didn't find much comments in the other files, I didn't add comments. I'll try to explain what I did to make the code easy to understand and maintainable. :slightly_smiling_face: